README.md

```diff
@@ -27,26 +27,17 @@ mvn package -DskipTests
 
 `mvn package` will assemble all the required dependencies and package into an uber jar.
 
-Pre-create Atlas models (Recommended)
-=====================================
+Create Atlas models
+===================
+NOTE: below steps are only necessary prior to Apache Atlas 2.1.0. Apache Atlas 2.1.0 will include the models. 
 
-You may need to create Atlas models to ensure all necessary models are created before running SAC and pushing metadata changes. Spark Atlas Connector provides the way to pre-create Atlas models, and it is only needed for the first time.
+You may need to create Atlas models for Spark to ensure all necessary models are created before running SAC and pushing metadata changes.
```

Comment:
Actually, just delete this line.

Reply:
OK. I assume Atlas end users would be familiar with its structure and know about models - then we can skip this line. Will reflect.